pkg/controller/master-controller-manager/kubeone/controller.go

```diff
@@ -59,8 +59,8 @@ import (
 )
 
 const (
-	// ControllerName is the name of this very controller.
-	ControllerName = "kubeone_controller"
+	// This controller is responsible for managing the lifecycle of KubeOne clusters within KKP.
+	ControllerName = "kkp-kubeone-manager"
```

Comment:
Why not `kkp-kubeone-controller`? It's not a `manager` afaik.

Reply:
Does that make a difference? What's a `manager` according to you? :)

Reply:
https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager this :D